src/main/java/Duke.java

```diff
@@ -1,10 +1,214 @@
+import models.Deadline;
+import models.Event;
+import models.Task;
+import models.Todo;
+import io.DukePrint;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+
+    private static final ArrayList<Task> tasks = new ArrayList<>();
+    private static int taskCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        DukePrint.printLogo();
+
+        Scanner sc = new Scanner(System.in);
```

Comment:
In general, it's best to use English words as variable names that makes it easier to tell what the variable is, especially later down in the code